projects/forms/README.md

```diff
@@ -1,25 +1,28 @@
 # Forms Typed (ngx-forms-typed)
+
 This project aims at providing several tools for Angular Forms development, including:
- - types for strongly typing your `FormControl`s, `FormGroup`s and `FormArray`s based on a Model type
- - functions for instantiating the `TypedFormControl`, `TypedFormGroup` and `TypedFormArray` based on a Model type
- - a helper function that enumerates all controls in a form (group)
-     - calls methods on each of them
-     - attaches their validators to a parent form (group)
-     - attaches the touched/untouched behavior to a parent form (group)
+
+- types for strongly typing your `FormControl`s, `FormGroup`s and `FormArray`s based on a Model type
+- functions for instantiating the `TypedFormControl`, `TypedFormGroup` and `TypedFormArray` based on a Model type
+- a helper function that enumerates all controls in a form (group)
+  - calls methods on each of them
+  - attaches their validators to a parent form (group)
+  - attaches the touched/untouched behavior to a parent form (group)
 
 ## Getting started
 
-1. `npm install --save-dev ngx-forms-typed`
+1. `npm i ngx-forms-typed`
```

Comment:
Looks great, can we please get rid of the rest of the changes - whitespaces, etc.?

Reply:
My bad. I forgot to turn off FormatOnSave.